package.json

```diff
@@ -125,6 +125,7 @@
     "sharp": "0.30.7",
     "tslib": "^2.3.1",
     "unleash-client": "^3.10.1",
+    "uuid": "^9.0.0",
```

Comment:
En ting jeg driver litt å lurer på her, har nav noen anbefalinger for hva man burde bruke? Eller er det v4() fra uuid de anbefaler? :blush:

Reply:
<img width="672" alt="image" src="https://user-images.githubusercontent.com/17828446/193253029-3e1976c0-1abe-4a63-af3a-8e1fdfe0d56b.png">